### DISCUSSION TITLE
OBS Websocket Password won't save error

### DISCUSSION THREAD

**Discussion by Jcsam6289** [Palakis/obs-websocket#832] [Q&A]
As stated in the title, inside OBS when i put in the new password it won't save and i can't connect to my Lioranboard, please help advice on how can i fix this please. Thank you.

**Comment by tt2468** [comment]
This sounds like user error. Have you tried copying your password to clipboard then using that to set the server and client password?

**Reply by Jcsam6289** [reply]
Thank you for replying, it started working out of nowhere again so i can't pinpoint what is causing it or how it happened, thank you for responding and replying tho

**Comment by SolarMisae** [comment]
not OP but I have done exactly that (copy/pasting it from a clipboard) but the password will not update no matter how many times I click okay, clean installs, reboots, etc, and just reverts to the 8 character password that it starts with and won't change to anything else. I'm sure there is something I'm missing but I can't seem to find it.

**Reply by tt2468** [reply]
4.x does not save the password itself so it always shows 8 dots, regardless of how long the real password is.

**Reply by SolarMisae** [reply]
Wow fast reply. And yeah. It still was failing to save but after my 5th reinstall a moment ago, it's now working. Thanks. :)
